libraries/src/Application/ApiApplication.php

```diff
@@ -144,8 +144,32 @@ public function addFormatMap($contentHeader, $format)
 	 */
 	protected function render()
 	{
+		// Trigger the onBeforeRender event.
+		PluginHelper::importPlugin('system');
+		$this->triggerEvent('onBeforeRender');
+
+		// Check we aren't in offline mode. In which case for users who can't access the site the API is disabled
+		// and we won't show anything!
```

Comment:
https://developer.joomla.org/coding-standards/inline-code-comments.html
```suggestion
		/**
		 * Check we aren't in offline mode. In which case for users who can't access the site the API is disabled
		 * and we won't show anything!
		 */
```